packages/gatsby-source-drupal/src/normalize.js

```diff
@@ -6,6 +6,7 @@ const nodeFromData = (datum, createNodeId) => {
     id: createNodeId(datum.id),
     drupal_id: datum.id,
     parent: null,
+    drupal_parent: attributes.parent,
```

Comment:
```suggestion
    drupal_parent_id: attributes.parent,
```

Reply:
Hey @sidharthachatterjee I think it needs to remain as drupal_parent because depending on  how Drupal has been setup it might not actually be an ID

Reply:
Can you list possible values types for this key?

Reply:
It's user-configurable so really depend on how the user configures it.